component/kyverno.jsonnet

```diff
@@ -62,6 +81,12 @@ local objects = [] +
                       generateSuccessEvents: params.generateSuccessEvents,
                     },
                   },
+
+                  kube.PodDisruptionBudget('kyverno') {
+                    spec+: {
+                      selector: deployment.spec.selector,
+                    } + params.podDisruptionBudget,
+                  },
```

Comment:
We probably want to avoid making a PDB if `replicas` = 1

Reply:
Don't know if that is trying to be too clever. Let's go for `enabled=true`?

Reply:
that, or maybe `minAvailable=0` does the same trick?

Reply:
> that, or maybe `minAvailable=0` does the same trick?

It does. Just tested it.